source/minimessage/format.rst

```diff
@@ -82,6 +82,33 @@ Examples
 .. image:: /minimessage/images/color_verbose_2.png
    :alt: The result of parsing ``<color:#FF5555>This is a <color:#55FF55>test!``, shown in-game in the Minecraft client's chat window
 
+Shadow Color
+************
+
+Color the shadow of the next parts
+
+Tag
+   * :mm:`<shadow:_hex_>`, for hex color
```

Comment:
`_hex_` supports alpha argument too if in `#RRGGBB` format (`#RRGGBBAA` takes priority over alpha arg)
Could maybe even combine both into `_colorNameOrHex_`